### ISSUE TITLE
[$10]Elements must have sufficient color contrast

### ISSUE THREAD

**Issue by kkartunov** [topcoder-platform/community-app#3091]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/community/accessibility-testing
2. Notice the menu elements and the copyright message

**Expected Result**
Expected contrast ratio of 4.5:1

**Actual Result**
Element has insufficient color contrast of 2.99 (foreground color: #959596, background color: #ffffff, font size: 10.5pt, font weight: normal)
Copyright message: Element has insufficient color contrast of 2.69 (foreground color: #6b6b6b, background color: #2a2a2b, font size: 9.0pt, font weight: normal).

**Device**: Desktop/Labtop
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: AA
**WCAG Success Criterion**: 1.4.3 Contrast (minimum)
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/msft-copilots_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=2521ab22%2D4cea%2D4606%2Da210%2D50280914399c&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fmsft%2Dcopilots%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2F%5BFAST%5D%20Test%20for%20Accessibility%20Series%2Fyoctocode%2Fbug%201

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097696 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097696 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by Dara-K** [comment]
Copyright menu color:  # 959596
Menu colors: # 555555

**Comment by kkartunov** [comment]
@topcoder-platform/topcodercompetitors this is open for pickup. PR against `qa-accessibility` branch.

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097696 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@r0hit-gupta  I guess this is completed by one of your PR?

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097696 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by r0hit-gupta** [comment]
Copyright Message - Fixed by PR https://github.com/topcoder-platform/community-app/pull/2932
Menu Items - Fixed by PR https://github.com/topcoder-platform/community-app/pull/3062

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```
